source.R

```diff
@@ -0,0 +1,156 @@
+	
+	if (length(cod1[1]) == 0 | length(pos_stop1+2) == 0) {pos1 <- "NONE"} else {pos1 <- dna[cod1[1]:(pos_stop1+2)]}
+	if (length(cod2[1]) == 0 | length(pos_stop2+2) == 0) {pos2 <- "NONE"} else {pos2 <- dna[cod2[1]:(pos_stop2+2)]}
+	if (length(cod3[1]) == 0 | length(pos_stop3+2) == 0) {pos3 <- "NONE"} else {pos3 <- dna[cod3[1]:(pos_stop3+2)]}
+	if (length(cod4[1]) == 0 | length(pos_stop4+2) == 0) {pos4 <- "NONE"} else {pos4 <- revcomp[cod4[1]:(pos_stop4+2)]}
+	if (length(cod5[1]) == 0 | length(pos_stop5+2) == 0) {pos5 <- "NONE"} else {pos5 <- revcomp[cod5[1]:(pos_stop5+2)]}
+	if (length(cod6[1]) == 0 | length(pos_stop6+2) == 0) {pos6 <- "NONE"} else {pos6 <- revcomp[cod6[1]:(pos_stop6+2)]}
+	
+	cat("ORF 1", file="output.txt", sep="\n")
+	cat(paste("Start:", cod1[1]), file="output.txt", append=TRUE, sep="\n")
+	cat(paste("Stop:", pos_stop1), file="output.txt", append=TRUE, sep="\n")
+	cat("Sequence: ", file="output.txt", append=TRUE)
+	cat(pos1, file="output.txt", append=TRUE)
+	cat(" ", file="output.txt", append=TRUE, sep="\n")
+
+	
+	cat("ORF 2", file="output.txt", append=TRUE, sep="\n")
+	cat(paste("Start:", cod2[1]), file="output.txt", append=TRUE, sep="\n")
+	cat(paste("Stop:", pos_stop2), file="output.txt", append=TRUE, sep="\n")
+	cat("Sequence: ", file="output.txt", append=TRUE)
+	cat(pos2, file="output.txt", append=TRUE)
+	cat(" ", file="output.txt", append=TRUE, sep="\n")
+	
+	cat("ORF 3", file="output.txt", append=TRUE, sep="\n")
+	cat(paste("Start:", cod3[1]), file="output.txt", append=TRUE, sep="\n")
+	cat(paste("Stop:", pos_stop3), file="output.txt", append=TRUE, sep="\n")
+	cat("Sequence: ", file="output.txt", append=TRUE)
+	cat(pos3, file="output.txt", append=TRUE)
+	cat(" ", file="output.txt", append=TRUE, sep="\n")
+
+	cat("ORF 4", file="output.txt", append=TRUE, sep="\n")
+	cat(paste("Start:", cod4[1]), file="output.txt", append=TRUE, sep="\n")
+	cat(paste("Stop:", pos_stop4), file="output.txt", append=TRUE, sep="\n")
+	cat("Sequence: ", file="output.txt", append=TRUE)
+	cat(pos4, file="output.txt", append=TRUE)
+	cat(" ", file="output.txt", append=TRUE, sep="\n")
+	
+	cat("ORF 5", file="output.txt", append=TRUE, sep="\n")
+	cat(paste("Start:", cod5[1]), file="output.txt", append=TRUE, sep="\n")
+	cat(paste("Stop:", pos_stop5), file="output.txt", append=TRUE, sep="\n")
+	cat("Sequence: ", file="output.txt", append=TRUE)
+	cat(pos5, file="output.txt", append=TRUE)
+	cat(" ", file="output.txt", append=TRUE, sep="\n")
+	
+	cat("ORF 6", file="output.txt", append=TRUE, sep="\n")
+	cat(paste("Start:", cod6[1]), file="output.txt", append=TRUE, sep="\n")
+	cat(paste("Stop:", pos_stop6), file="output.txt", append=TRUE, sep="\n")
+	cat("Sequence: ", file="output.txt", append=TRUE)
+	cat(pos6, file="output.txt", append=TRUE)
+	cat(" ", file="output.txt", append=TRUE, sep="\n")
+
+	return(cat(longest))
+
+}
```

Comment:
When I run your code, the output is not correct -- I get this written to output.txt, but these sequences are not in sampleDNA.csv if you look at the base pair numbers:

Start: 265
Stop: 
Sequence: NONE 
ORF 2
Start: 338
Stop: 380
Sequence: A T G C T C A A G C A A G G T G C G G C G A T G A T T G G G G A T C G A A C G C C G T A A 
ORF 3
Start: 123
Stop: 210
Sequence: A T G G A A G C G T C A C C T T A T T T T C T G A G C A C C G A G C G A T T A G A A G T G T C A A A C G C T T A T C T T C C C A A A C G A A A C G A A A G T C T G C T C C G A T A G 
ORF 4
Start: 400
Stop: 
Sequence: NONE 
ORF 5
Start: 173
Stop: 323
Sequence: A T G A A T G G T A G A G C T T C C T A T A A T T C A G T A G G C T C G G G G C T C C A C C T G A A G C G C A G A C C A A C A T T T A A C G A T G A C C G A A T T A G G A C T C T T C T A A T C C T T G T G G G T T G G C T A C G C A T C T A T C G G A G C A G A C T T T C G T T T C G T T T G G G A A G A T A A 
ORF 6
Start: 72
Stop: 108
Sequence: A T G A G A A C G G A T A T T C T G T G C A C A A C G T T T G T T T G C T A G 

However, when I look at your Rmarkdown, there are some correct ORFs in there. What happened?